backend/core/src/shared/email/email.service.ts

```diff
@@ -49,15 +49,15 @@ export class EmailService {
         )}...`
       )
     }
-    await this.send(
+    ;-(await this.send(
```

Comment:
This `;-` seems wrong here. Is there something missing? Should this be within the curly braces?